Matveibos/hometask-4/my_array_rotate.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# the module extends the class Array
+module Addition
```

Comment:
Module name should speak for methods it provides. Addition is not what this module provides, let's name it `LabArrayRotate`, or something like that